inbox/models/message.py

```diff
@@ -288,67 +288,56 @@ def create_from_synced(
 
         Parameters
         ----------
-        mid : int
+        imap_uid : int
             The account backend-specific message identifier; it's only used for
             logging errors.
 
-        body_string : bytes
+        body : bytes
             The full message including headers (encoded).
 
         """
-        _rqd = [account, mid, folder_name, body_string]
-        if not all([v is not None for v in _rqd]):
-            raise ValueError(
-                "Required keyword arguments: account, mid, folder_name, body_string"
-            )
-
         # stop trickle-down bugs
         assert account.namespace is not None
-        assert isinstance(body_string, bytes)
+        assert isinstance(body, bytes)
 
-        msg = Message()
+        message = Message()
 
-        msg.data_sha256 = sha256(body_string).hexdigest()
+        message.data_sha256 = sha256(body).hexdigest()
 
         # Persist the raw MIME message to disk/ S3
-        save_to_blockstore(msg.data_sha256, body_string)
+        save_to_blockstore(message.data_sha256, body)
 
         # Persist the processed message to the database
-        msg.namespace_id = account.namespace.id
+        message.namespace_id = account.namespace.id
 
         try:
-            body_length = len(body_string)
+            body_length = len(body)
             if body_length > MAX_MESSAGE_BODY_LENGTH:
                 raise MessageTooBigException(body_length)
-            parsed: MimePart = mime.from_string(body_string)
-            # Non-persisted instance attribute used by EAS.
-            msg.parsed_body = parsed
```

Comment:
This is only used in `get_headers()` below, that is not used. So I just deleted it.